projects/unicode-rs/unicode-segmentation-fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,11 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+use unicode_segmentation::UnicodeSegmentation;
+
+fuzz_target!(|data: &[u8]| {
+    if let Ok(s) = std::str::from_utf8(data) {
+        let _g = s.graphemes(true).collect::<Vec<&str>>();
```

Comment:
for this and split_word_bounds perhaps we can assert that it is the same as the original string?

Reply:
Sounds good to me! We have the fuzzers upstream now so let's do the changes there!